components/picker/index.css

```diff
@@ -10,492 +10,501 @@ OF ANY KIND, either express or implied. See the License for the specific languag
-  --spectrum-picker-font-color-hover: var(--spectrum-neutral-content-color-hover);
-  --spectrum-picker-font-color-hover-open: var(--spectrum-neutral-content-color-focus-hover);
-  --spectrum-picker-font-color-active: var(--spectrum-neutral-content-color-down);
-  --spectrum-picker-font-color-key-focus: var(--spectrum-neutral-content-color-key-focus);
-
-  --spectrum-picker-icon-color-default: var(--spectrum-neutral-content-color-default);
-  --spectrum-picker-icon-color-default-open: var(--spectrum-neutral-content-color-focus);
-  --spectrum-picker-icon-color-hover: var(--spectrum-neutral-content-color-hover);
-  --spectrum-picker-icon-color-hover-open: var(--spectrum-neutral-content-color-focus-hover);
-  --spectrum-picker-icon-color-active: var(--spectrum-neutral-content-color-down);
-  --spectrum-picker-icon-color-key-focus: var(--spectrum-neutral-content-color-key-focus);
-
-  --spectrum-picker-border-color-error-default: var(--spectrum-negative-border-color-default);
-  --spectrum-picker-border-color-error-default-open: var(--spectrum-negative-border-color-focus);
-  --spectrum-picker-border-color-error-hover: var(--spectrum-negative-border-color-hover);
-  --spectrum-picker-border-color-error-hover-open: var(--spectrum-negative-border-color-focus-hover);
-  --spectrum-picker-border-color-error-active: var(--spectrum-negative-border-color-down);
-  --spectrum-picker-border-color-error-key-focus: var(--spectrum-negative-border-color-key-focus);
-
-  --spectrum-picker-icon-color-error: var(--spectrum-negative-visual-color);
-
-  --spectrum-picker-background-color-disabled: var(--spectrum-disabled-background-color);
-  --spectrum-picker-font-color-disabled: var(--spectrum-disabled-content-color);
-  --spectrum-picker-icon-color-disabled: var(--spectrum-disabled-content-color);
-
-  /* special cases for focus indicator */
-  --spectrum-picker-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
-  --spectrum-picker-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
-  --spectrum-picker-focus-indicator-color: var(--spectrum-focus-indicator-color);
+	/* Font */
+	--spectrum-picker-font-size: var(--spectrum-font-size-100);
+	--spectrum-picker-font-weight: var(--spectrum-regular-font-weight);
+	--spectrum-picker-placeholder-font-style: var(--spectrum-default-font-style);
+	--spectrum-picker-line-height: var(--spectrum-line-height-100);
+	--spectrum-picker-line-height-cjk: var(--spectrum-cjk-line-height-100);
+
+	/* Dimensions */
+	--spectrum-picker-block-size: var(--spectrum-component-height-100);
+	--spectrum-picker-inline-size: var(--spectrum-field-default-width-medium);
+	--spectrum-picker-min-inline-size: calc(var(--spectrum-picker-minimum-width-multiplier) * var(--mod-picker-block-size, var(--spectrum-picker-block-size)));
+
+	/* Border */
+	--spectrum-picker-border-radius: var(--spectrum-corner-radius-medium-size-medium);
+	--spectrum-picker-border-width: var(--spectrum-border-width-100);
```

Comment:
I don't think this was really called out in the designs, but with some furious double clicking in Figma, I see the invalid border, is 2px. So should this be `--spectrum-border-width-200`? 

Also! I saw Rise had a great idea to [update the copyright year](https://github.com/adobe/spectrum-css/blob/7a77ce5f9d54dc3d0698f1583d445b6c28c431ee/components/menu/index.css#L2) at the top of the file. I'm pretty sure I did in status light, so just wanted to share that with you too in case you hadn't seen it. 😊

Reply:
Good callout. I had asked about the border width during design review, because it's missing from the Figma spec, but nothing was said about it. I just followed up about this question specifically and it should use `--spectrum-border-width-200`. I've updated this in the latest push.